packages/js/src/ui/components/Inbox.tsx

```diff
@@ -1,17 +1,19 @@
-import { JSX } from 'solid-js';
+import { createSignal, JSX, Match, Switch } from 'solid-js';
 import { useAppearance, useLocalization } from '../context';
 import { cn, useStyle } from '../helpers';
 import { Bell } from './Bell';
 import { Footer } from './Footer';
-import { Header } from './Header';
+import { Header, SettingsHeader } from './Header';
 import { Popover, popoverContentClasses, popoverTriggerClasses } from './Popover';
+import { Settings } from './Settings';
 
 type InboxProps = {
   open?: boolean;
   renderBell?: ({ unreadCount }: { unreadCount: number }) => JSX.Element;
 };
 
 export const Inbox = (props: InboxProps) => {
+  const [isSettingsOpen, setIsSettingsOpen] = createSignal(false);
```

Comment:
Can we do `string` instead of `boolean` here. Like `currentScreen` that has values `inbox` or `settings` ?